morph_tool/utils.py

```diff
@@ -51,6 +66,9 @@ def neurondb_dataframe(filename: Path) -> pd.DataFrame:
         df = pd.DataFrame(data=rows, columns=['name', 'layer', 'mtype', 'use_axon'])
```

Comment:
does use_axon have the right value according to the (strange) rules?

Reply:
I have added a test for that.